pdc_client/bin/pdc_client

```diff
@@ -173,7 +173,7 @@ if __name__ == "__main__":
     data = load_data(options)
 
     for key, value in data.iteritems():
-        if options.request == 'GET' and not value:
+        if options.request == 'GET' and not value and value.lower() is not 'false':
```

Comment:
I think it may cause exception if value is a empty list or dict. Maybe last condition just "value is not False" or "value != False" should be fine. String "false" will not pass 'not value' check. It should not be the condition need to be ruled out.